verify_test_dsa.go

```diff
@@ -1,182 +0,0 @@
-// +build go1.11 go1.12 go1.13 go1.14 go1.15
```

Comment:
This line was apparently intended to compile the test only on Go 1.11-1.15.
However, this was actually compiled for every Go (>= 1.11) releases.

The test was not executed by `go test` because the file does not have `_test.go` suffix.